bin/atnodes

```diff
@@ -308,6 +312,8 @@ OPTIONS:
     -v            Be verbose.
     -w            Prompt for password (used for login and sudo,
                   could be privided by SSH_BATCH_PASSWORD).
+    -W            Prompt for password (like -w but conflict, just for sudo.
+                  Never use -W together with -w, because -w will be ignored).
```

Comment:
@JamesPan Thanks for the hard work! Regarding the CLI design patterns, I just follow common conventions and my intuition :)